pkg/ostree/ostree.go

```diff
@@ -83,6 +134,10 @@ func VerifyRef(ref string) bool {
 	return len(ref) > 0 && ostreeRefRE.MatchString(ref)
 }
 
+func VerifyChecksum(commit string) bool {
```

Comment:
nitpick: This function and the one above could be private, no?

Reply:
Yup.  We can make them public if we need to later.